### DISCUSSION TITLE
Add explosive capabilities to E2

### DISCUSSION THREAD

**Discussion by DecVes** [wiremod/wire#2641] [Suggestions]
make it so that any prop or entity can be explosive with like PropExplosive(Number Radius,Number Timeafterexplosion,Number TImebeforeexplosion) etc
then you can trigger it using something like Prop:explode(String) or smth like that Prop:explode()

also if you add this PLEASE make it so the radius is infinite i hate the radius on normal wire explosive

**Comment by Denneisk** [comment]
The convar `wire_explosive_range` controls maximum radius of the Wire explosive.
This might work better as part of some sort of combat extension(?) and have a `createExplosion` function instead.

**Comment by vurvdev** [comment]
This can be done with extending the `damage` extension to add a `blastDamage` function like what damagecore has now.

> also if you add this PLEASE make it so the radius is infinite i hate the radius on normal wire explosive

lolno

**Comment by Hazeofdream** [comment]
> > also if you add this PLEASE make it so the radius is infinite i hate the radius on normal wire explosive
> 
> lolno

server convar, ez pz

**Comment by DecVes** [comment]
> This can be done with extending the `damage` extension to add a `blastDamage` function like what damagecore has now.
> 
> > also if you add this PLEASE make it so the radius is infinite i hate the radius on normal wire explosive
> 
> lolno

hey i got a question did you guys implement the damage issue in e2 yet?

**Reply by vurvdev** [reply]
Damage getters exist yes, so you can run when damage is dealt. Not pushed to workshop yet but I can push soon

**Reply by DecVes** [reply]
yes please push the workshop version i have the workshop version! but i do have a question if i get the wiremod github vrsion can other people see my new wiremod contraptions?

**Reply by vurvdev** [reply]
The server needs to be on github. Your local install of wire doesn't matter

**Comment by vurvdev** [comment]
Resolved by https://github.com/wiremod/wire/pull/2675
